### PR TITLE
WIP: SA-76: Ensure PoA and Document steps can be used together

### PR DESCRIPTION
**Pull request by dlewton**
# Problem

When a user chooses to use the Document step along with the PoA step, the data from one overwrites the other in the current development. Specifically, only the data from one of these steps is sent through to the onComplete callback.

# Solution

Rework the code so that the PoA step generates a separate key in the onComplete callback. This should ensure that the data from either step can be accessed within the onComplete callback.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [yes] Has the CHANGELOG been updated?
- [yes] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the TESTING_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [?] Have any new automated tests been implemented or the existing ones changed?
- [?] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -194,6 +194,7 @@
     "postcss-custom-media": "^7.0.8",
     "postcss-loader": "^3.0.0",
     "postcss-url": "^8.0.0",
+    "pre-commit": "^1.2.2",
```
Comment (it-ony) [inline package.json:197]: why is pre-commit added again? We just got rid of it.

File: src/components/utils/redux.ts [hunk 2]
```diff
@@ -9,7 +9,9 @@ export const buildCaptureStateKey = (payload: {
 }): Exclude<keyof CaptureState, 'takesHistory'> => {
   const { method, side, variant } = payload
 
+  if (method === 'poa') return 'poa'
```
Comment (it-ony) [inline src/components/utils/redux.ts:12]: let me suggest a map here

Comment (dlewton) [reply src/components/utils/redux.ts:12]: I'm not sure I understand how to implement the suggestion of a map here. Currently, only the values of `poa` and `face` directory correspondent to a `method`. But, other methods depend on the values of `side` and `variant`.

**Comment by dlewton** [comment]
An attempt to correct issues with pre-commit affecting my PR codebase through rebasing turned this PR into a mess. The best option forward appears to be to re-create a cleaner PR.

**Pull request closed without merging**